### DISCUSSION TITLE
Scale factor event

### DISCUSSION THREAD

**Discussion by ThatsNoMoon** [hecrj/iced#1306] [Ideas]
Hi hi, enjoying using `iced` so far.

It would be nice if `iced` would include an event like `winit`'s [`ScaleFactorChanged`](https://docs.rs/winit/latest/winit/event/enum.WindowEvent.html#variant.ScaleFactorChanged). Since `iced` seems to handle different scale factors pretty well already, I imagine it wouldn't be too difficult to also send that event to the user. My current use-case is drawing a different icon based on scale factor, as I've found the smaller sizes of the icon need to have higher contrast to stay as legible. Other considerations might include getting the scale factor before the window is open, if possible.

**Comment by hecrj** [comment]
Sounds reasonable!

It should be fairly easy to add a new `window::Event` and trigger it from our shells.
